pages/implementation/supported-programming-languages.mdx

```diff
@@ -4,7 +4,7 @@ title: 'Languages'
 # abstract: 'Learn how to use Avo to create a tracking plan and generate high quality, actionable data.'
```

Comment:
So many opportunities here!

I'm wondering if we should merge Tools and Reference? I kind of feel like this page could live under Reference at least, what do you think?

Reply:
Not sure about merging, but moving programming languages to reference makes sense.

Reply:
How do we connect it to the Avo Functions though, should we have it as a "child" of Avo Functions in Reference?

Reply:
Added a section for avo functions